src/Twig/TwigAssetsEngine.php

```diff
@@ -70,6 +71,7 @@ class TwigAssetsEngine
      * - cache_lifetime: Default is 0
      * - cache_path: The temporary cache path
      * - path: The public assets cache directory (e.g. public/cache)
+     * - base_path: The absolute path of the minified css/js.
```

Comment:
The name `base_path` sounds like a "real" path from the filesystem (and it must not be absolute), but it's more url specific base path. Other people could be confused because of the existing `path` setting. Maybe we should rename it to `url_base_path` to be more clear about the intention / usage.